test/clt-tests/optimisation-and-update/test-optimisation-and-update.rec

```diff
@@ -0,0 +1,154 @@
+––– block: ../base/start-searchd –––
+––– input –––
+apt-get update -y > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+apt-get install -y bc > /dev/null; echo $?
+––– output –––
+debconf: delaying package configuration, since apt-utils is not installed
+0
+––– input –––
+chmod +x ./test/clt-tests/scripts/load_names_attr.php > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+php -d memory_limit=2G ./test/clt-tests/scripts/load_names_attr.php 100000 4 2000000 1 2G 1 2 async > /dev/null; echo $?
```

Comment:
min_infix_len='2' - intentionally? (wasn't in the MRE)

Reply:
Fixed